docs/deployment/deploy_manually.md

```diff
@@ -355,6 +355,8 @@ The following procedures are performed on a MySQL client. You must have MySQL cl
 
    If the field `Alive` is `true`, this CN node is properly started and added to the cluster.
 
+   After CNs are properly started, you need to set the system variables `SET prefer_compute_node = true;` and `SET use_compute_nodes = -1;` to scale computing resources during queries. For more information, see [System variables](../reference/System_variable.md#descriptions-of-variables).
```

Comment:
After CNs are properly started and you want to use CNs during queries, set the system variables `SET prefer_compute_node = true;` and `SET use_compute_nodes = -1;`.